src/components/language-switcher/component.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { hot } from 'react-hot-loader';
+import { connect } from 'react-redux';
+import cx from 'classnames';
+import { tx } from '@transifex/native';
+import { useLanguages, useLocale } from '@transifex/react';
+import styles from './language-switcher.module.scss';
+
+const mapStateToProps = ({ location }) => ({
+  route: location.routesMap[location.type],
+});
+
+const LanguageSwitcher = (props) => {
+  const { route } = props;
+  const { page } = route;
+  const languages = useLanguages();
+  const locale = useLocale();
+
+  const renderLanguageSwitcher = () => languages.map(({ code }, index) => (
+    <button
+      className={
+        cx(
+          styles.languageButton,
+          locale === code && styles.languageButtonSelected,
```

Comment:
If we don't have a language selected we should highlight 'en' as it is the default